source/_docs/wordpress-cron.md

```diff
@@ -45,7 +45,7 @@ You can also schedule your own jobs, execute existing jobs, and manage just abou
 One of the first things you'll want to do is test WP-Cron to make sure everything is working correctly. When you execute the command below, make sure to replace SITE_NAME with your site's name from your Pantheon Dashboard and replace ENV_NAME with the desired environment ("dev", "test", "live", or multidev branch name).
 
 ````nohighlight
-$ terminus wp <site>.<env> "cron test"
+$ terminus wp <site>.<env> cron test
```

Comment:
This should be `terminus wp <site>.<env> -- cron test`

The `--` can be anywhere, and is not necessary at all if there are no options. However, I have my eye on using it as a hint to disambiguate <site>.<env> from the next parameter. Never a problem for wp-cli (site.env must always have a `.`, and no wp-cli commands contain a `.`), but it may be important in other instances, so let's start training users early.